meal-planning/src/components/recipe-card/RecipeCard.js

```diff
@@ -58,6 +72,31 @@ const RecipeCard = ({ recipe }) => {
           </IconButton>
         </CardActions>
       </Card>
+
+
+      <div>
+        <Dialog
+          open={open}
+          onClose={handleCloseNo}
+          aria-labelledby="alert-dialog-title"
+          aria-describedby="alert-dialog-description"
+        >
+          <DialogTitle id="alert-dialog-title">{"Delete the recipe for " + recipe.name +"?"}</DialogTitle>
+          <DialogContent>
+            <DialogContentText id="alert-dialog-description">
+              The recipe will be permanently deleted.
+            </DialogContentText>
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={handleCloseNo} color="primary">
+              No, thanks!
+            </Button>
+            <Button onClick={handleCloseYes} color="primary" autoFocus>
+              Yes, please!
+            </Button>
+          </DialogActions>
+        </Dialog>
+      </div>
```

Comment:
This should be its own component. see AddRecipeDialog as an example. Move all the dialog stuff into a new file and control the state in this file.